src/core/xfa/template.js

```diff
@@ -2266,7 +2266,7 @@ class Image extends StringObject {
         c.charCodeAt(0)
       );
```

Comment:
Somewhat unrelated, but I cannot help wonder why we're sort-of "reinventing the wheel" rather than using the existing `stringToBytes` utility function (from `src/shared/util.js`) here?

Reply:
Just because the wheel is so small here but you're right I'll replace it.